src/firefly/js/visualize/ui/ImageTableRowViewer.jsx

```diff
@@ -64,7 +142,7 @@ export function ImageTableRowViewer({viewerId, makeRequestFromRow, defaultCutout
         <MultiImageViewer
             {...{viewerId, Toolbar, insideFlex, defaultImageCnt, maxImageCnt, tableId:tbl_id,
                 makeRequestFromRow,defaultCutoutSizeAS,
-                wcsMatchType, activePlotId,
+                wcsMatchType, activePlotId, makeCustomLayout: makeImageSlider,
```

Comment:
if you move `makeImageSlide` out this will need to become something liek
```js
makeCustomLayout: (viewerItemIds, makeItemViewer) => makeImageSlider(viewerItemIds, makeItemViewer,table,viewerId, etc)
```